### DISCUSSION TITLE
How can I use relative url to import?

### DISCUSSION THREAD

**Discussion by lemonwidz** [quasarframework/quasar#10217] [General - Components / Directives / etc] [answered]
import test from '../../plugins/common/test.js'

this is works fine. but..

import test from '@plugins/common/test.js'

this is not working.

I wanna to write all path with relative.

How can I do?

--- THIS IS ERROR MESSAGE --------
Module not found: Can't resolve imported dependency "@plugins/common/test.js"
Did you forget to install it? You can run: npm install --save @plugins/common/test.js

**Comment by metalsadman** [comment, marked as answer]
@lemonwidz see https://quasar.dev/quasar-cli/handling-webpack#webpack-aliases

**Reply by lemonwidz** [reply]
@metalsadman I already tried.

            chainWebpack(chain) {
                chain
                    .plugin('eslint-webpack-plugin')
                    .use(ESLintPlugin, [{ extensions: ['js', 'vue'] }])
                chain.resolve.alias.set(
                    'plugins',
                    path.resolve(__dirname, './src/plugins')
                )
            }

But not working.

**Reply by metalsadman** [reply]
plugins, not @plugins

**Reply by lemonwidz** [reply]
@metalsadman WOW! Great! It works. Thank so much.

**Reply by rstoenescu** [reply]
Or use current aliases. Like `src` for `src/plugins/....`
